.github/workflows/on_pull_request.yml

```diff
@@ -151,60 +103,25 @@ jobs:
         with:
           name: compiled-js-and-types
           path: packages/
+          if-no-files-found: error
```

Comment:
ℹ️ I noticed that if the save-cache step _fails_ it just throws a warning, and does not error out. This will cause the job to fail and the action to stop running if no files are found on download. (also added to the job `build_website`